setup.cfg

```diff
@@ -8,6 +8,7 @@ install_requires =
     boto3==1.26.43
     pandas==1.5.2
     pydantic==1.10.4
+    pytest-mock==3.10.0
```

Comment:
This isn't an application dependency, and instead should go under the test dependencies below without a pinned version.